.gitignore

```diff
@@ -14,5 +14,3 @@ website/apidocs/
 *.ipr
 *.iws
 .DS_Store
-server/src/main/resources/keywhiz-development.yaml
```

Comment:
We might want to keep this if people want to configure their dev keywhiz and avoid accidentally committing it.

Reply:
We have a keywhiz-development.yaml in the repo now, though.

Previously we copied this from a mysql/h2 specific version in maven, which is why it was ignored